corehq/form_processor/models.py

```diff
@@ -174,7 +177,7 @@ def is_submission_error_log(self):
         return self.state == self.SUBMISSION_ERROR_LOG
 
     @property
-    @memoized
+    @cache_return_value
```

Comment:
i have mixed feelings on this (see comment below, though could be misunderstanding something). i feel like our abstraction is wrong if we're trying to do this. i think something further up the chain should be cached. in this case, maybe `get_attachment` should be `cached` based on the `attachment_name`. 

in the function below, we should have a util function that takes a `last_sync_token` and returns the sync_log and that function should be cached based on the `last_sync_token`

let me know what you think